sagemaker/launch/launch_realtime_endpoint.py

```diff
@@ -0,0 +1,269 @@
+"""
+    Copyright Contributors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Launch a SageMaker endpoint for realtime inference
+"""
+
+import os
+import argparse
+import logging
+import tempfile
+import boto3 # pylint: disable=import-error
+from time import gmtime, strftime
+import sagemaker as sm
+from botocore.exceptions import WaiterError
+
+from launch_utils import (wrap_model_artifacts,
+                          check_tarfile_s3_object,
+                          parse_s3_url,
+                          upload_data_to_s3,
+                          check_name_format)
+
+
+# TODO: When adding new realtime inference tasks, modify this list
+SUPPORTED_REALTIME_INFER_TASKS = ['node_classification']
+
+# Constants for SageMaker endpoints
+ENTRY_FOLDER_NAME = 'realtime_entry_points'
+ENTRY_FILE_NAMES = [
+    'node_classification_entry.py',
+    'node_regression_entry.py',
```

Comment:
Do we have these files? If not, please remove them.
We can add later.

Reply:
removed. And only keep the `node_classification_entry.py` for this PR.